tests/testthat/test_cdfPlot.R

```diff
@@ -2,34 +2,35 @@ context("DGEobj.plots - tests for cdfPlot.R functions")
 
 
 test_that("cdfPlot.R: cdfPlot()", {
-    skip_if(!("ReplicateGroupDesign_fit" %in% names(t_obj1)))
-
-    top_table <- topTable(t_obj1$ReplicateGroupDesign_fit, number = 100)
+    skip_if(!("BDL_vs_Sham" %in% names(t_obj1)))
 
+    DGEdata = t_obj1
+    contrast = "BDL_vs_Sham"
     # testing plot with default values.
-    plot <- cdfPlot(top_table, referenceLine = "blue")
+    plot <- cdfPlot(DGEdata, contrast, referenceLine = "blue")
```

Comment:
For all test cases , specify both the parameter name and the value . You are specifying only the value for the first couple of parameters.